### PR TITLE
Support requests bugfix

### LINKED ISSUE #227: [Bug] Support Request endpoint is not storing the JSON data in the DB
**Describe the bug**
Support Request endpoint is storing the JSON data `person` and `support_data` in the DB as `null`

**To Reproduce**

1. Send the following rest request

```shell
curl --request POST \
  --url http://localhost:5000/api/v1/support-request \
  --header 'Content-Type: application/json' \
  --data '{
	"person": {
		"email": "test@example.com",
		"name": "John Doe",
		"phone": "+35983333333333",
		"comment": "test",
		"terms": true,
		"gdpr": true,
		"newsletter": true
	},
	"support_data": {
		"roles": {
			"benefactor": false,
			"partner": false,
			"associationMember": true,
			"company": false,
			"volunteer": false
		},
		"benefactor": {
			"campaignBenefactor": false,
			"platformBenefactor": false
		},
		"partner": {
			"npo": false,
			"bussiness": false,
			"other": false,
			"otherText": ""
		},
		"volunteer": {
			"backend": false,
			"frontend": false,
			"marketing": false,
			"designer": false,
			"projectManager": false,
			"devOps": false,
			"security": false,
			"financesAndAccounts": false,
			"lawyer": false,
			"qa": false
		},
		"associationMember": {
			"isMember": true
		},
		"company": {
			"sponsor": false,
			"volunteer": false,
			"other": false,
			"otherText": ""
		}
	}
}'
```

2. See new uuid in the response
3. Fetch records from the db

```shell
docker exec -it podkrepi-roach-0 bash
cockroach sql --insecure --host podkrepi-roach-0 --database app
```

```sql
select * from app.support_requests;
```

4. See `null` in columns `person` and `support_data`

**Expected behavior**

See the newly created record with all fields stored in the db

**Screenshots**

image

### PR DESCRIPTION
**Pull request by imilchev**
Fixes #227

### CODE REVIEW

**Comment by imilchev** [comment]
Since there are no workflows configured for this module and no tests, I propose to re-write it. Perhaps even move it to a separate repo and give it, its own database. I am willing to do that if people agree that it is a good idea. @kachar ?

**Comment by kachar** [comment]
@imilchev Go for it

As for the database, currently the db `app` is only used by this module, so we can say it's dedicated for it

File: backend/contact/routes/support_request/person_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const personPayload = `{
+	"address": "Sofia",
```
Comment (kachar) [inline backend/contact/routes/support_request/person_test.go:11]: @imilchev Let's add a field `comment` as it's a mandatory part of the payload now

File: backend/contact/routes/support_request/person.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+func (j *Person) Scan(value interface{}) error {
+	return json.Unmarshal([]byte(fmt.Sprintf("%v", value)), &j)
+}
+
+type Person struct {
+	Email      string `json:"email" valid:"required,email"`
+	Name       string `json:"name" valid:"required,minstringlength(2)~field-too-short,maxstringlength(50)~field-too-long"`
+	Phone      string `json:"phone" valid:"required,phone"`
+	Address    string `json:"address"`
```
Comment (kachar) [inline backend/contact/routes/support_request/person.go:22]: Let's add a field `Comment` `string` as it's mandatory part of the frontend payload now

**Comment by kachar** [comment]
@imilchev I've tested the update locally and What I see is that we miss the `person.comment` in the responses although it's sent in the creation request.

**Comment by imilchev** [comment]
@stanimirdim92 started this so kudos to him as well

**Comment by imilchev** [comment]
Linting for graphql-gateway and module-accounts fails because of the protoc generated files but didn't have time to investigate it further

**Comment by stanimirdim92** [comment]
I have included a forgotten GDPR field.

**Review by kachar — APPROVED** [review]

**Pull request merged**